packages/algob/src/builtin-tasks/compile.ts

```diff
@@ -25,9 +29,11 @@ function compileTask ({ force }: TaskArgs, env: RuntimeEnv): Promise<void> {
 
 export async function compile (force: boolean, op: CompileOp): Promise<void> {
   await assertDir(CACHE_DIR);
+  const paths = lsTreeWalk(ASSETS_DIR);
 
-  for (const f of readdirSync(ASSETS_DIR).sort(cmpStr)) {
-    if (!f.endsWith(tealExt) && !f.endsWith(pyExt)) { continue; }
+  for (const p of paths.sort(cmpStr)) {
+    const f = path.basename(p);
+    if ((!f.endsWith(tealExt) && !f.endsWith(pyExt)) || p.includes(ALGOBPY_DIR)) { continue; }
```

Comment:
we should update the documentation, that files in `/assets/algobpy` are ignored when looking for smart contract files.

Reply:
ok